### ISSUE TITLE
The command create-package and create-profile is not working on Windows 10

### ISSUE THREAD

**Issue by norman-ingal** [redcanari/canari3#33]
After building and compiling version 3.2.1 successfully, I tried issuing the command ``canari create-package test``. The directory containing the package is empty.
```shell
c:\maltego>canari create-package test
creating skeleton in test
--> Project description: test

--> Author name [admin]: name

--> Author email: email@gmail.com

done!

c:\maltego>dir test
 Volume in drive C has no label.
 Volume Serial Number is EEEE-EEEE

 Directory of c:\maltego\test

04/19/2018  08:25 AM    <DIR>          .
04/19/2018  08:25 AM    <DIR>          ..
04/19/2018  08:25 AM               228 .mrbob.ini
               1 File(s)            228 bytes
               2 Dir(s)   4,728,356,864 bytes free
```

Also tried to import a custom package by the command `canari create-profile ThreatGrid`:

```
c:\maltego\ThreatGrid\src>canari create-profile ThreatGrid
Looking for transforms in ThreatGrid...
Package loaded.
Creating profile c:\maltego\ThreatGrid\src\ThreatGrid.mtz...
Installing transform ThreatGrid.TGQueryHash from ThreatGrid.transforms.query_hash.TGQueryHash...
Traceback (most recent call last):
  File "C:\Python27\Scripts\\canari", line 4, in <module>
    __import__('pkg_resources').run_script('canari==3.2.1', 'canari')
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 658, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 1438, in run_script
    exec(code, namespace, namespace)
  File "c:\python27\lib\site-packages\canari-3.2.1-py2.7.egg\EGG-INFO\scripts\canari", line 25, in <module>
    canari_main()
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\commands\framework.py", line 33, in _func
    return func(self.parser.parse_args(args_, namespace))
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\commands\common.py", line 67, in canari_main
    opts.command_function(opts)
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\commands\create_profile.py", line 60, in create_profile
    transform_package.create_profile(opts.working_dir, current_dir)
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\pkgutils\transform.py", line 368, in create_profile
    self.install(install_prefix, mtz, configure)
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\pkgutils\transform.py", line 293, in install
    self._install_transforms(install_prefix, distribution)
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\pkgutils\transform.py", line 302, in _install_transforms
    distribution.add_transform(prefix, 'Local', transform, server='Local')
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\pkgutils\maltego.py", line 349, in add_transform
    transform_settings_def
  File "C:\Python27\lib\site-packages\canari-3.2.1-py2.7.egg\canari\pkgutils\maltego.py", line 707, in write_file
    contents = contents.render(fragment=True, pretty=True)
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\__init__.py", line 419, in render
    data.extend(self._render(nsmap))
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\__init__.py", line 496, in _render
    for child in children:
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\__init__.py", line 532, in _render_children
    for data in datas:
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\fields.py", line 620, in render_children
    for data in chunks:
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\fields.py", line 603, in child_chunks
    for data in self.field.render_children(obj, item, nsmap):
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\fields.py", line 485, in render_children
    for data in val._render(nsmap):
  File "C:\Python27\lib\site-packages\safedexml-0.5.1-py2.7.egg\safedexml\__init__.py", line 502, in _render
    raise RenderError("Field '%s' is missing" % (f.field_name,))
safedexml.RenderError: Field 'value' is missing
Writing transform set Threatgrid to c:\maltego\ThreatGrid\src\ThreatGrid.mtz...
Writing server Local to c:\maltego\ThreatGrid\src\ThreatGrid.mtz
```

The custom package was created in Mac OSX environment, then copied to Windows 10 machine.

**Comment by allfro** [comment]
Could you check if this is also the case in py3 under Windows?

**Comment by norman-ingal** [comment]
I’m using python 2.7.14

**Comment by allfro** [comment]
Should be fixed in the latest PR #34
